app/product/[handle]/page.tsx

```diff
@@ -131,13 +131,13 @@ async function RelatedProducts({ id }: { id: string }) {
                   currencyCode: product.priceRange.maxVariantPrice.currencyCode
                 }}
                 src={product.featuredImage?.url}
-                width={600}
-                height={600}
+                fill
```

Comment:
Are you missing `object-fit`?

Reply:
No. `object-contain` is *inside* of `GridTileImage`.